apis/fluentd/v1alpha1/fluentd_types.go

```diff
@@ -86,6 +86,8 @@ type FluentdSpec struct {
 	VolumeClaimTemplates []corev1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"`
 	// Service represents configurations on the fluentd service.
 	Service FluentDService `json:"service,omitempty"`
+	// FSGroup represents the fsGroup that will be used in the fluentd PodSecurityContext.
+	FSGroup *int64 `json:"fsGroup,omitempty"`
```

Comment:
@erhudy Thanks for the contribution.

I think we need to add `SecurityContext` field here just like fluentbit does:
https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/fluentbit_types.go#L77

Reply:
@benjaminhuo updated, sorry for the delay.